Python/Tests/ReplWindowUITests/ReplWindowSendUITests.cs

```diff
@@ -159,24 +166,101 @@ ...     time.sleep(5)
             );
         }
 
+        /// <summary>
+        /// Verifies the workspace interpreter is used.
+        /// </summary>
+        public void SendToInteractiveWorkspaceInterpreter(PythonVisualStudioApp app) {
+            RunFromWorkspace(app, "PrintInterpreter.py",
+                Input("import sys").Complete,
+                Input("print(sys.version_info[:2])").Complete.Outputs("(3, 7)"),
+                EndOfInput
+            );
+        }
+
+        /// <summary>
+        /// Verifies the workspace package can be imported.
+        /// </summary>
+        public void SendToInteractiveWorkspacePackage(PythonVisualStudioApp app) {
+            RunFromWorkspace(app, "ImportWorkspacePackage.py",
+                Input("import package1").Complete,
+                Input("print(package1.PACKAGE_MSG)").Complete.Outputs("Package Success"),
+                EndOfInput
+            );
+        }
+
+        /// <summary>
+        /// Verifies the workspace search path package can be imported.
+        /// </summary>
+        public void SendToInteractiveWorkspaceSearchPathPackage(PythonVisualStudioApp app) {
+            RunFromWorkspace(app, "ImportSearchPathPackage.py",
+                Input("import folder1").Complete,
+                Input("print(folder1.SEARCH_PATH_MSG)").Complete.Outputs("Search Path Success"),
+                EndOfInput
+            );
+        }
+
         #endregion
 
         #region Helpers
 
-        private void RunOne(PythonVisualStudioApp app, string filename, params SendToStep[] inputs) {
-            // SendToInteractive.pyproj uses Python 3.5 32-bit
-            var settings = ReplWindowSettings.FindSettingsForInterpreter("Python35");
-            var sln = app.CopyProjectForTest(@"TestData\SendToInteractive.sln");
-            var project = app.OpenProject(sln);
-            var program = project.ProjectItems.Item(filename);
-            var window = program.Open();
+        private void RunFromProject(PythonVisualStudioApp app, string filename, params SendToStep[] inputs) {
+            PythonPaths.Python27_x64.AssertInstalled();
+            PythonPaths.Python37_x64.AssertInstalled();
```

Comment:
This will fail if the user does not have python 27 and 37 installed. Is there some way to look for the interpreters the user has installed and use that?

Reply:
In this case, user = a ptvs developer = us, and python 2.7 and 3.7 are the bare minimum needed/expected to be installed in order to run tests. The AssertInstalled is so we have a nice error if we try to run the test and forgot to install the correct CPython.